dev-libs/zziplib/zziplib-0.13.59.recipe

```diff
@@ -1,31 +1,95 @@
-DESCRIPTION="
-Lightweight library used to easily extract data from files archived in a \
-single zip file.
-"
+SUMMARY="The zziplib extract data from archives in a single zip"
+DESCRIPTION="Lightweight library used to easily extract data from \
+files archived in a single zip file."
 HOMEPAGE="http://zziplib.sourceforge.net/"
+COPYRIGHT="1999-2010 Guido Draheim"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
 SOURCE_URI="http://sourceforge.net/projects/zziplib/files/zziplib13/0.13.59/zziplib-0.13.59.tar.bz2/download"
```

Comment:
> @fbrosson Changes done.

I can see only the first change, the one for COPYRIGHT (2009 -> 2010), but not the second suggeston in my previous [review](#pullrequestreview-16947995) for SOURCE_URI, which was to change SOURCE_URI to
```
SOURCE_URI="https://downloads.sourceforge.net/zziplib/zziplib-$portVersion.tar.bz2"
```
or
```
SOURCE_URI="https://downloads.sf.net/zziplib/zziplib-$portVersion.tar.bz2"
```

BTW, I don't understand why GitHub has already hidden the comment. Normaly comments in reviews only become hidden (by default) if the line to which they were "attached" changes. But this does not seem to be the case here.
Well, I guess this will remain unexplained ;-) But that's not important, after all.
What's important is that we have a nice SOURCE_URI that uses SourceForge's redirector service (i.e. that we use one of the URIs I suggested ;-)